codegen/gateway_exec.go

```diff
@@ -33,8 +33,7 @@ func ExecWithFS(ctx context.Context, cln *client.Client, fs Filesystem, opts Opt
 		cwd = fs.Image.Config.WorkingDir
 	}
 
-	env := make([]string, len(fs.Image.Config.Env))
-	copy(env, fs.Image.Config.Env)
+	env = append(env, fs.Image.Config.Env...)
```

Comment:
It might be a good idea to make a copy of `env` before appending to it. It may not be safe to append to a slice that's passed in and might be reused for other purposes.